src/configs/haskell.rs

```diff
@@ -0,0 +1,23 @@
+use crate::config::{ModuleConfig, RootModuleConfig, SegmentConfig};
+
+use ansi_term::{Color, Style};
+use starship_module_config_derive::ModuleConfig;
+
+#[derive(Clone, ModuleConfig)]
+pub struct HaskellConfig<'a> {
+    pub symbol: SegmentConfig<'a>,
+    pub version: SegmentConfig<'a>,
+    pub style: Style,
+    pub disabled: bool,
+}
+
+impl<'a> RootModuleConfig<'a> for HaskellConfig<'a> {
+    fn new() -> Self {
+        HaskellConfig {
+            symbol: SegmentConfig::new("λ "),
+            version: SegmentConfig::default(),
```

Comment:
I still haven't worked with the new config style but why is the `version` part of the `Config` when it's not configurable?

Reply:
See #472